scripts/GLTFModelBuilder.js

```diff
@@ -50,12 +57,17 @@ async function loadEntityIds(cache, options, configType, modelType) {
             return;
         }
         let model = await cache.getDef(IndexType.MODELS, entityDef[modelType]);
+        individualModels.push(model);
+        individualModelNames.push(entityDef.name ?? model.id.toString());
         finalModel.addModel(model);
+        // save where the model vertices start for the corresponding model
+        modelVertexIndices.push(i === 0 ? 0 : modelVertexIndices[i - 1] + individualModels[i - 1].vertexPositionsX.length);
     }
 }
 
 async function exportGLTFModel(cache) {
     const exporter = new GLTFExporter(finalModel.getMergedModel());
+    const exporters = individualModels.map((m) => new GLTFExporter(m));
```

Comment:
this is a bit wasteful (we create and populate the exporters  even if split is false, and just don't do anything with them) but maybe when split === false we can just push a single exporter with the mergedModel to exporters